src/Symfony/Component/Form/FormErrorIterator.php

```diff
@@ -265,6 +266,16 @@ public function seek($position)
         }
     }
 
+    public function getFormErrorByCode($code)
```

Comment:
What about `hasErrorCode`? And what if there are many errors with the same code?

I suggest either return a boolean: false => no such error, true => error(s), or null/array of error(s).